File: packages/alteration/src/main/java/fdit/alteration/core/incident/Altitude.java
package fdit.alteration.core.incident;

import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlText;

public class Altitude {

    @JacksonXmlProperty(isAttribute = true)
    private Boolean offset;

    @JacksonXmlProperty(isAttribute = true)
    private Boolean time;

    @JacksonXmlText()
    private Integer content;

    public Boolean getOffset() {
        if (offset == null) {
            return false;
        }
        return offset;
    }

    public void setOffset(final Boolean offset) {
        this.offset = offset;
    }

    public Boolean getTime() {
        if (time == null) {
            return false;
        }
        return time;
    }

    public void setTime(Boolean time) {
        this.time = time;
    }

    public Integer getContent() {
        if (content == null) {
            return 0;
        }
        return content;
    }

    public void setContent(final Integer content) {
        this.content = content;
    }
}

File: packages/fdit-ts/src/Incident/Altitude.ts
export class Altitude {
  private offset?: boolean
  private time?: boolean
  private content?: number

  public getOffset(): boolean {
    return this.offset ?? false
  }

  public setOffset(offset: boolean): void {
    this.offset = offset
  }

  public getTime(): boolean {
    return this.time ?? false
  }

  public setTime(time: boolean): void {
    this.time = time
  }

  public getContent(): number {
    return this.content ?? 0
  }

  public setContent(content: number): void {
    this.content = content
  }
}
